docs/application/vscode-ext/web.md

```diff
@@ -240,11 +305,51 @@ To run project:
 To debug application, follow these steps:
 
 1. In VS Code, click **View** > **Command Palette**.
-2. In the input field of the **Command Palette** that appears, type **Run Web Simulator** and press **Enter**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Web Simulator** and press **Enter**.
 3. After the **Web Simulator** is launched, enter the path of the file in the box. For example, file:///C://Workspace/basic/index.html.
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
+### Run Design Editor
+
+To run design editor, follow these steps:
+
+> **Note** 
+> 
+> **The user must select the html extension file in the file explorer.**
+>
+> There are `3` ways to launch the Design Editor.
+
+- Command Palette
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+3. **Design-editor** launches while creating a new tab.
+
+- Click **button** of **status bar**
+1. Run by clicking the Design-Editor button at the bottom right of the screen.
+
+    ![Design Editor button](media/tizen_web27.png)
+
+- Rignt click preference
+1. Select the html file to launch and right click.
+2. Click **Tizen Web: Run Design-Editor** from the list.
+
+    ![Design Editor right click](media/tizen_web28.png)
+
+This image has successfully launched the Design Editor.
+
+![Design Editor launch](media/tizen_web29.png)
+
+### Quick Action Execution
+Right-click on the project folder to use the extension function in the menu.
+
+Please check the previous contents for a detailed guide of each function.
```

Comment:
Please give a reference link here and use the format:

For more information on the functions, see "link".

Reply:
done.